src/components/pages/AdminHome/ApplicationCard.js

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import { Card } from 'antd';
+import { connect, useDispatch } from 'react-redux';
+import { getInstructors } from '../../../redux/actions/instructorActions';
+import { useOktaAuth } from '@okta/okta-react';
+import '../../../styles/AdminDashboardHome/index.less';
+
+const ApplicationCard = props => {
+  const history = useHistory();
+  const dispatch = useDispatch();
+  const { authState } = useOktaAuth();
+
+  useEffect(() => {
+    if (!authState) return;
+    dispatch(getInstructors(authState.idToken.idToken));
+  }, [authState]);
+
+  const handleOnClick = id => {
+    history.push(`/instructors/${id}`); // Might have to change the route.
+  };
+
+  const sorted = props.instructor.instructors.sort(
+    (a, b) => new Date(b.date) - new Date(a.date)
+  );
+  const firstTen = sorted.slice(0, 10);
+
+  return firstTen.map(app => {
+    return (
+      <Card
+        style={{
+          width: '100%',
+          backgroundColor: '#FCDFAA',
+          marginBottom: '2%',
+        }}
+      >
+        <p>
+          Name: <span>{app.name}</span>
```

Comment:
I'm guessing the `<span>` is used to add space between the label and value. If not, do we need it?

Reply:
We added the span so we could get rid of the bolding as the card is originally bolded